src/jitlayers.cpp

```diff
@@ -187,7 +187,7 @@ void NotifyDebugger(jit_code_entry *JITCodeEntry)
     }
     __jit_debug_descriptor.first_entry = JITCodeEntry;
     __jit_debug_descriptor.relevant_entry = JITCodeEntry;
-    __jit_debug_register_code();
+    //__jit_debug_register_code();
```

Comment:
Oops, sorry, I made sure to leave that out originally, but when I fixed the indent, it snuck back. Will fix.